src/07-even-more-loops.js

```diff
@@ -44,7 +61,15 @@ function shoutWhileLoop(array) {}
  *  shoutForOfLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutForOfLoop(array) {}
+function shoutForOfLoop(array) {
+  const arr3 = []
+ for (let ele of array){
+  arr3.push(ele + "!");
+ }
+
+
+  return arr3;
+}
```

Comment:
Keep scope in mind—there is no need for an `arr3` name here, as there's no `arr2` in this scope.

You might also give them different names based on what they are. `arr3` could be `appendedStrings` or `shouts` or `result` or `answer`. Whatever name makes it so that you never have to think, "What is in that array, again?"